src/encoded/static/components/search.js

```diff
@@ -708,6 +728,7 @@ const DatasetComponent = (props, reactContext) => {
                                         )
                                     </span>
                                 : null}
+                                {examineLociGene && examineLociGene.length > 0 && fccSeries ? <span>{` targeting ${examineLociGene.join(', ')}`} </span> : null}
```

Comment:
I feel like this line and the one on line 716 would be easier to read if the ternary was spread over separate lines. I think most of the code in Encode follows that convention.

Reply:
ok, fixed